mindmeld/components/entity_recognizer.py

```diff
@@ -54,7 +54,6 @@ def __init__(self, resource_loader, domain, intent):
         self.domain = domain
         self.intent = intent
         self.entity_types = set()
-        self._model_config = None
 
```

Comment:
Looks like there is no much use to have this additional variable which we were previously dumping and loading. Happy to take suggestions on this change. Removing this variable necessitates changes to one unittest in _test\_nlp.py_ file. We do not have this variable saved for RoleClassifiers, wondering what could be the use-case for EntityReconizers.

Reply:
In the config, you can specify what file name patterns you want to train the model on. For example, in the case of Webex Assistant, the call_person intent is trained on one subset of files whereas the entity recognizer for that intent is trained on a larger subset of files. As the test in `test_nlp.py` suggests, the config is dumped so that we have access to the actual train files to extract queries from so we can decipher whether it is an incremental build or not. 

I actually think we should be dumping the config for all models. This will help reproduce results since one can change train/test file patterns, feature sets, hyperparam values etc while building an application. So dumping the config along with the model will make things more maintainable.

Reply:
Looks like we can remove it.